src/test/java/org/jabref/logic/formatter/minifier/TruncateFormatterTest.java

```diff
@@ -0,0 +1,36 @@
+package org.jabref.logic.formatter.minifier;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * Tests in addition to the general tests from {@link org.jabref.logic.formatter.FormatterTest}
+ */
+public class TruncateFormatterTest {
+    private final String TITLE = "A Title";
+
+    @Test
+    void truncateModifierWorksWith0Length() {
```

Comment:
maybe also add a test with invalid data and negative values

Reply:
I'll move it back to draft for now. I need a test for `createDirNameFromPattern` and modifications to deal with stacked folders.